### DISCUSSION TITLE
error in init_reaction: datamdbuilder is stuck.

### DISCUSSION THREAD

**Discussion by caojiachun** [deepmodeling/dpgen#1360] [Q&A]
The datamdbuilder program got stuck after processing part of the structure and could not process the rest. Could this be due to a possible structural problem reported by Open babel? 

This is the part of build.log:

==============================
*** Open Babel Warning  in PerceiveBondOrders
  Failed to kekulize aromatic bonds in OBMol::PerceiveBondOrders

Read trajectory: 61966timestep [00:36, 3217.81timestep/s]==============================
*** Open Babel Warning  in PerceiveBondOrders
  Failed to kekulize aromatic bonds in OBMol::PerceiveBondOrders

Read trajectory: 100001timestep [00:55, 1814.23timestep/s]
2023-10-11 00:52:07,163 - MDDatasetBuilder 1.3.8 - INFO: Step 1 Done! Time consumed (s): 55.919
C1111:   0%|                                                                                                                                                                                    | 0/100001 [00:00<?, ?timestep/s]
C1111: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100001/100001 [34:15<00:00, 48.65timestep/s]
2023-10-11 01:26:23,754 - MDDatasetBuilder 1.3.8 - INFO: Max counter of C1111 is Counter({'H': 20, 'C': 19, 'O': 17})
C112:   0%|                                                                                                                                                                                     | 0/100001 [00:00<?, ?timestep/s]

**Comment by caojiachun** [comment]
I tested the other lammpstrj, and there seems to be the wrong counter.

![image](https://github.com/deepmodeling/dpgen/assets/71744229/5da60472-1249-4de4-8bc3-f192ef1c096b)

**Reply by qijingwei199** [reply]
Did you resolve the issues?can  i get your wechat to disscuss?
